packages/data-context/src/sources/ProjectDataSource.ts

```diff
@@ -101,6 +102,40 @@ export function transformSpec ({
   }
 }
 
+export function getDefaultSpecFileName (specPattern: string, fileExtensionToUse?: 'js' | 'ts') {
+  function formatGlob (s: string, fallback: string) {
+    let [cleanedVal] = s.replace('{', '').replace('}', '').split(',')
```

Comment:
I'm kind of surprised this works - how come we don't need either `replaceAll` or `replace('{', //g)`? I see you have a test with mutiple `{` and `}` and it works just fine, but I don't see anywhere that handles replacing multiple `{` in a single glob. Did I miss something?

Reply:
Test I was referring to with multiple `{`: https://github.com/cypress-io/cypress/pull/20654/files#diff-59bf8023d3f96e2d91bc479be5372ebde0317cf6040e5e514ceb276ac71c1342R221